draft-ietf-dots-requirements.md

```diff
@@ -243,7 +243,7 @@ communication channel between DOTS agents. Indeed, establishing a relationship
 with peer DOTS agents during normal network conditions provides the foundation
 for more rapid attack response against future attacks, as all interactions
 setting up DOTS, including any business or service level agreements, are already
-complete.
+complete. Peer DOTS agents are provisioned to a DOTS client using a variety of means (e.g., manual or dynamic). 
```

Comment:
Can we reword to something like: `Peer DOTS agents are provisioned to a DOTS client using a variety of manual or dynamic methods`?